src/main/java/net/stickycode/plugin/kuuty/KuutyGenerateSecretMojo.java

```diff
@@ -16,9 +16,9 @@
     extends AbstractMojo {
 
   /**
-   * Directory containing the config template files to encapsulate
+   * Directory containing the secret sample files to encapsulate
```

Comment:
containing the secret template files? why sample? slip of the mind or intentional? If intentional you'll need to explain to me.

Reply:
Intentional, the output is a secret template in my common usage so the input can't be the template, I settled on sample in my head. But suggest a better name.